openhands/agenthub/codeact_agent/codeact_agent.py

```diff
@@ -107,12 +108,7 @@ def __init__(
             f'TOOLS loaded for CodeActAgent: {json.dumps(self.tools, indent=2, ensure_ascii=False).replace("\\n", "\n")}'
         )
         self.prompt_manager = PromptManager(
-            microagent_dir=os.path.join(
-                os.path.dirname(os.path.dirname(openhands.__file__)),
-                'microagents',
-            )
```

Comment:
This needs to stay--this is loading the microagents built into _this_ repo, i.e. the public microagents